scss/grid/_column.scss

```diff
@@ -16,37 +16,7 @@
 ///
 /// @returns {Number} A calculated percentage value.
 @function grid-column($columns) {
-  $width: 0%;
-
-  // Parsing percents, decimals, and column counts
-  @if type-of($columns) == 'number' {
-    @if unit($columns) == '%' {
-      $width: $columns;
-    }
-    @else if $columns < 1 {
-      $width: percentage($columns);
-    }
-    @else {
-      $width: percentage($columns / $grid-column-count);
-    }
-  }
-
-  // Parsing "n of n" expressions
-  @else if type-of($columns) == 'list' {
-    @if length($columns) != 3 {
-      @error 'Wrong syntax for grid-column(). Use the format "n of n".';
-    }
-    @else {
-      $width: percentage(nth($columns, 1) / nth($columns, 3));
-    }
-  }
-
-  // Anything else is incorrect
-  @else {
-    @error 'Wrong syntax for grid-column(). Use a number, decimal, percentage, or "n of n".';
-  }
-
-  @return $width;
+  @return fraction-to-percentage($columns, $denominator: $grid-columns);
```

Comment:
Because of this the sass tests fail when you run `test:sass` (precisely `test/sass/_components.scss`)

```
Error: Undefined variable; "$grid-columns".
```

What do you think is the best way to fix it?
Just adding `$grid-columns: 12;` to the top of the test file?

Reply:
All options are already defined inside components. For the standard grid, it is defined in `scss/grid/_grid.scss`. So we have this error because I miscalled the grid option, I should use `$grid-column-count` instead.

Reply:
All options are already defined inside components. For the standard grid, it is defined in `scss/grid/_grid.scss`. So we have this error because I miscalled the grid option, I should use `$grid-column-count` instead.

Reply:
Btw I just seen that we do not run the Sass tests in Travis 😄

Reply:
ok I see, then using `$grid-column-count` in `_column.scss` is definitely the right fix

Reply:
@SassNinja Fixed